content/200-concepts/200-database-connectors/03-postgresql.mdx

```diff
@@ -127,13 +128,15 @@ Note that `localhost` is required, the value itself is ignored and can be anythi
 
 > **Note**: You can find additional context in this [GitHub issue](https://github.com/prisma/prisma-client-js/issues/437#issuecomment-592436707).
 
-## Type mapping between PostgreSQL to Prisma schema
+## Type mapping between PostgreSQL and Prisma schema
 
-The PostgreSQL connector maps the [scalar types](/concepts/components/prisma-schema/data-model#scalar-fields) from the Prisma [data model](/concepts/components/prisma-schema/data-model) as follows to native column types:
+These two tables show the type mapping between PostgreSQL and Prisma schema. First [how Prisma scalar types are translated into PostgreSQL database column types](TODO), and then [how PostgreSQL database column types relate to Prisma scalar and native types](TODO).
```

Comment:
what are these hyperlinks supposed to link to? the two tables themselves? using two such long hyperlinks in a short paragraph makes the paragraph difficult to read IMO, maybe we can find a better way to link out to the resources you want to link to?


<img width="419" alt="image" src="https://github.com/prisma/docs/assets/4058327/01aca0d0-ec27-422a-90cb-a8fa73a767d0">

Reply:
Yes, absolutely open for suggestions. This will also be mirrored on the other pages, so we should definitely find a good way. (The "how" can totally be left out of the link, but that won't help much I guess)

Reply:
```suggestion
These two tables show the type mapping between PostgreSQL and Prisma schema. First [how Prisma scalar types are translated into PostgreSQL database column types](#mapping-between-prisma-scalar-types-and-postgresql-database-column-types), and then [how PostgreSQL database column types relate to Prisma scalar and native types](#mapping-between-postgresql-database-column-types-to-prisma-scalar-and-native-types).
```